.github/workflows/node-ci.yml

```diff
@@ -221,6 +221,9 @@ jobs:
         run: |
           brew install docker docker-compose
 
+      - name: Make sure we have enough room
```

Comment:
I guess this is the relevant change for the docker build failure. If so, can you remove the other files from this pull request?

Reply:
@wipfli Let us see if we can merge https://github.com/maplibre/maplibre-gl-native/pull/630 first. With that the files would automatically be in sync.

Reply:
Merge did the magic. Now it is one file only!